eng/common/generate-locproject.ps1

```diff
@@ -91,6 +91,7 @@ $locJson = @{
             )
         },
         @{
+            LanguageSet = $LanguageSet
```

Comment:
Shouldn't this change be in Arcade?

Reply:
I'm rushing a bit because a change on the loc side (the `WiX_CloneLanguages` change mentioned in the description) broke the current `aspnetcore-localization` pipeline and caused the messed-up https://github.com/dotnet/aspnetcore/commit/e5329489516b214e24017037716cabf0e1edba1a commit. That commit adds incorrectly-mapped files like src/Installers/Windows/SharedFrameworkBundle/cs-CZ/thm.wxl.

I'm about to put up a PR for Arcade making exactly the same fix but want tomorrow's `OneLocBuild` task to work cleanly.